src/vaibhav/SecondMax.java

```diff
@@ -0,0 +1,28 @@
+package vaibhav;
+
+/*Assignment - 29 : 26th Nov'2020 
+Find second maximum from array without sorting the array.*/
+
+public class SecondMax {
+	
+	public static void getSecondElement(int[] array1) {
+		int len = array1.length;
+		for (int index = 0 ; index < len;index++) {
```

Comment:
Sorting logic is correct but in sorting you are using two loops here,
you can hanld in single loop
Check this code out
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/nandini/SecondMaxNumber.java